pyproject.toml

```diff
@@ -1,58 +1,47 @@
-# Copyright (C) 2022 Obscurity Labs LLC. <admin@obscuritylabs.com> - All Rights Reserved
-# Unauthorized copying of this file, via any medium is strictly prohibited.
-# All rights reserved. No warranty, explicit or implicit, provided.
-# Proprietary and confidential.
-
 [tool.poetry]
-authors = ["Obscurity Labs"]
-description = "Project DarkMoon - Web API service"
 name = "darkmoon"
-version = "0.0.1"
+version = "0.0.2"
+authors = ["Obscurity Labs"]
+description = "Project DarkMoon - CLI tool"
+readme = "README.md"
 
 [tool.poetry.scripts]
 darkmoon = "darkmoon.main:cli_app"
 
 [tool.poetry.dependencies]
-MarkupSafe = "2.0.0"
-aenum = "^3.1.11"
-asyncio = "^3.4.3"
-fastapi = {version = "^0.95.0", extras = ["all"]}
-fastapi-mail = "^0.4.1"
-loguru = "^0.5.3"
-pydantic = {extras = ["dotenv"], version = "^1.8.2"}
-pydantic-jsonapi = "^0.11.0"
-pyjwkest = "^1.4.2"
 python = "^3.10"
-python-dotenv = "^0.15.0"
-python-slugify = "^4.0.1"
+fastapi = {version = "^0.95.0", extras = ["all"]}
```

Comment:
Do you really need all the extras in FastAPI? There are a lot of really expensive and not well maintained dependencies brought in here that aren't in use in this codebase.